crates/turborepo-scm/src/git.rs

```diff
@@ -354,10 +354,17 @@ impl GitRepo {
         let stdout = String::from_utf8(stdout).unwrap();
         for line in stdout.lines() {
             let path = RelativeUnixPath::new(line).unwrap();
-            let anchored_to_turbo_root_file_path = self
-                .reanchor_path_from_git_root_to_turbo_root(turbo_root, path)
-                .unwrap();
-            files.insert(anchored_to_turbo_root_file_path);
+            match self.reanchor_path_from_git_root_to_turbo_root(turbo_root, path) {
+                Ok(anchored_to_turbo_root_file_path) => {
+                    files.insert(anchored_to_turbo_root_file_path);
+                }
+                Err(err) => {
```

Comment:
Bubble up this error so it reaches the caller and they can decide what to do with it. In our case when this error is eventually returned in `change_detector.rs` we should catch it and behave as if every package changed since we cannot identify which exact packages changed. The warning you added should be moved there with an updated message.